python_modules/libraries/dagster-airflow/dagster_airflow/operators/operator_to_op.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+from contextlib import contextmanager
+from typing import List, Optional
+
+from airflow.models import Connection
+from dagster import check, op, Any, In, Nothing, OpDefinition, Out
+
+
+@contextmanager
+def replace_airflow_logger_handlers(handler):
+    try:
+        root_logger = logging.getLogger()
+        root_logger.addHandler(handler)
+        yield
+    finally:
+        root_logger.removeHandler(handler)
+
+
+def operator_to_op(
```

Comment:
What do you think about calling this `airflow_operator_to_op` to make it clearer?

Reply:
Yes, I think that's better